crates/uv-resolver/src/lock.rs

```diff
@@ -146,8 +140,8 @@ impl Lock {
                     else {
                         continue;
                     };
-                    let marker = edge.weight().as_ref();
-                    locked_dist.add_dependency(dependency_dist, marker);
+                    let marker = edge.weight().clone();
```

Comment:
Do we need to clone here in order to support unwrapping to default?

Reply:
We used to clone inside `add_dependency` anyways. Also a `MarkerTree` is just a `usize` now, it could even implement `Copy`.

Reply:
👍 Yeah sorry saw that afterwards, they're all interned now, right?

Reply:
Yeah.